src/base/static/components/templates/right-sidebar.js

```diff
@@ -2,20 +2,33 @@ import React from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
 
-import { Button } from "../atoms/buttons";
+import styled from "react-emotion";
 import StoryNavigator from "../organisms/story-navigator";
 import MapLegendPanel from "../organisms/map-legend-panel";
 import ActivityStream from "../organisms/activity-stream";
 
 import { rightSidebarConfigSelector } from "../../state/ducks/right-sidebar-config";
+import { setMapSizeValidity } from "../../state/ducks/map";
 
 import "./right-sidebar.scss";
 
+const ToggleSidebarButton = styled("div")({
+  cursor: "pointer",
+});
+
 const RightSidebar = props => {
   // TODO: Support multiple simultaneous right sidebar components.
   return (
     <div className="right-sidebar">
-      <Button className="right-sidebar__collapse-btn" />
+      {props.rightSidebarConfig.component !== "ActivityStream" && (
```

Comment:
Actually I think this check is not needed-- there was some css in `pbgreensboro`'s `custom.css` that was causing the button not to render.

I will open a PR with a fix. I tried using the new code insertion feature but I think since the change I need to make is in a file that's not part of this diff I can't use it in this case. ☹️

Reply:
haha good to know